src/app/components/ActiveUserLoanContainer/components/ActiveLoanTableContainer/index.tsx

```diff
@@ -112,14 +112,14 @@ export function ActiveLoanTableContainer(props: Props) {
         ),
         maintenanceMargin: stringToPercent(item.maintenanceMargin, 2),
         actions: (
-          <div className="d-flex flex-row flex-nowrap justify-content-end">
+          <div className="tw-flex tw-flex-row tw-flex-nowrap tw-justify-end">
             <div className="mr-1">
               <Tooltip
                 content={t(translations.activeLoan.table.container.topUp)}
               >
                 <Icon
                   icon="double-chevron-up"
-                  className="text-green mr-1 rounded-circle border border-green p-1"
+                  className="text-green mr-1 tw-rounded-full-circle border border-green p-1"
```

Comment:
I believe these should have tw- prefix
```suggestion
                  className="text-green mr-1 tw-rounded-full-circle tw-border-1 tw-border-green tw-p-1"
```